public/entry/entry.pug

```diff
@@ -292,7 +293,9 @@
                 li.list-group-item(ng-repeat="e in entry.sources") 
                   a(ng-click="search({sources: e.abbrev})",
                     ng-bind-html="highlighted('sources', e.abbrev)"
-                  ) 
+                  )
+                  a.mentioned-names-popup
```

Comment:
Sources look good. Text looks blue, though -- is that what we want?

![image](https://user-images.githubusercontent.com/1689183/66275907-5e5ab800-e842-11e9-92da-f212f09213bc.png)


Compare this to the mentioned names popup:
![image](https://user-images.githubusercontent.com/1689183/66275913-6ca8d400-e842-11e9-864d-15ed710f32b8.png)